azurerm/internal/services/azurestackhci/parse/hci_cluster_test.go

```diff
@@ -0,0 +1,72 @@
+package parse
```

Comment:
This one should also be replaced by the generated one.

Reply:
Could you share me an example? Thanks.

Reply:
updated